### PR TITLE
docs(k8s): updated version information

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Updated K8s version information. Do not merge before go.

### CODE REVIEW

**Review by RoRoJ — APPROVED** [review]

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 1]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
 | 1.21               | April 8, 2021    | June 2022            | April 30, 2021               | April 2023       | June 15, 2023     |
 | 1.22               | August 4, 2021   | October 2022         | August 11, 2021              | August 2023      | October 15, 2023  |
 | 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
-| 1.24               | April 2022       |                      |                              |                  |                   |
+| 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
+| 1.25               | August 2022      | October 2023         | February 1, 2023             | February 2025    | March 15, 2025    |
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:51]: release date moved to February 6

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 2]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.26               | December 2022    | February 2024        | February 1, 2023             | February 2025    | March 15, 2025    |
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:52]: release date moved from February 1 to February 6

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 3]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
 * Exec Probe Timeout Handling
 
 ### 1.19
-Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
\ No newline at end of file
+Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
+
+## Supported Container Runtime Interfaces (CRI)
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:107]: Similar to this section, we do need another ## section below this one, named Supported Container Network Interfaces (CNI)

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 4]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+
+### Kubernetes v1.26
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:109]: this ### section is not needed

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 5]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+Starting from Kubernetes version 1.26 launch:
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:110]: 1.25

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 6]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+* Kubernetes Kapsule only supports **cilium** and **calico** as CNI.
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:111]: This whole part refers to the CNI, not the CRI.
Lines 111 to 122 need to move under 
## Supported Container Network Interfaces (CNI)

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 7]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+* Kubernetes Kosmos only supports **kilo** as CNI.
+
+| Interface | Deprecation |  Obsolete  |
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:114]: - We can remove the Deprecation column, as we make Flannel & Weave become obsolete straight from 1.25.
- Let's put 1.25 in the obsolete column.

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 8]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+|-----------|-------------|------------|
+| Flannel   | from v1.25  | from v1.26 |
+| Weave     | from v1.25  | from v1.26 |
+
+* Migration policy:
+  * Control Planes can not be upgraded on obsolete CNIs.
+  * *Recommended action:* Create new clusters using a supported CNI, either via API or from the Scaleway console.
+
+### Kubernetes v1.25
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:123]: this ### is not needed

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 9]
```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
...
+Starting from Kubernetes version 1.25 launch, **containerd** is the only runtime supported by Scaleway Kubernetes products.
+
+| Runtime | Deprecation | Obsolete    |
+|---------|-------------|-------------|
+| Docker  | from v1.23  | from v1.24  |
+| Cri-o   | from v1.24  | from v.1.25 |
```
Comment (tgenaitay) [inline containers/kubernetes/reference-content/version-support-policy.mdx:129]: typo "from v.1.25" => from v1.25

**Review by tgenaitay — COMMENTED** [review]
Thanks for the draft!
Big watch out is that CRI != CNI.
Let's separate the two topics in ## sections, starting from CRI, then CNI.

**Review by tgenaitay — APPROVED** [review]
LGTM 👍

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 10]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 15, 2025    |
 
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation lists only the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
 
+### 1.26
+* Official [release announcement|https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/]
+*Main changes:*
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/version-support-policy.mdx:61]:
```suggestion
**Main changes:**
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 11]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 15, 2025    |
 
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation lists only the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
 
+### 1.26
+* Official [release announcement|https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/]
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/reference-content/version-support-policy.mdx:60]:
```suggestion
* Official [release announcement](https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/)
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 12]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+**Main changes:**
+* Eleven features graduated to GA
+* Twelve features deprecated or removed
+
+### 1.25
+ * Official [release announcement|https://kubernetes.io/blog/2022/08/23/kubernetes-v1-25-release/]
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/reference-content/version-support-policy.mdx:66]:
```suggestion
 * Official [release announcement](https://kubernetes.io/blog/2022/08/23/kubernetes-v1-25-release/)
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 13]
```diff
@@ -47,13 +47,26 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+ *Main changes:*
+ * Thirteen enhancements graduated to GA (see announcement for full list)
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/reference-content/version-support-policy.mdx:68]: link to announcement?

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 14]
```diff
@@ -89,4 +102,35 @@ The following documentation lists only the main feature changes for each version
...
+* Kubernetes Kapsule only supports **cilium** and **calico** as CNI.
+* Kubernetes Kosmos only supports **kilo** as CNI.
+
+| Interface |   Obsolete  |
+|-----------|-------------|
+| Flannel   |  from v1.25 |
+| Weave     |  from v1.25 |
+
+* Migration policy:
+  * Control Planes can not be upgraded on obsolete CNIs.
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/reference-content/version-support-policy.mdx:119]:
```suggestion
  * Control Planes cannot be upgraded on obsolete CNIs.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 15]
```diff
@@ -89,4 +102,35 @@ The following documentation lists only the main feature changes for each version
...
+
+Starting from Kubernetes version 1.25 launch, **containerd** is the only runtime supported by Scaleway Kubernetes products.
+
+| Runtime | Deprecation | Obsolete    |
+|---------|-------------|-------------|
+| Docker  | from v1.23  | from v1.24  |
+| Cri-o   | from v1.24  | from v1.25 |
+
+* Migration policies:
+  * Pools using deprecated CRIs can not scale up.
```
Comment (nedaehm) [inline containers/kubernetes/reference-content/version-support-policy.mdx:133]:
```suggestion
  * Pools using deprecated CRIs cannot scale up.
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 16]
```diff
@@ -89,4 +102,35 @@ The following documentation lists only the main feature changes for each version
...
+  * Control Planes can not be upgraded if CRI used by any pool is obsolete.
```
Comment (nedaehm) [inline containers/kubernetes/reference-content/version-support-policy.mdx:134]:
```suggestion
  * Control Planes cannot be upgraded if CRI used by any pool is obsolete.
```

**Pull request merged**